examples/MicroserviceExample/WebApi/RabbitMqService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+using RabbitMQ.Client;
+
+namespace WebApi
+{
+    public class RabbitMqService
+    {
+        private const string QueueName = "TestQueue";
+
+        private readonly ILogger<RabbitMqService> logger;
+        private readonly MessageSender messageSender;
+        private readonly ConnectionFactory connectionFactory;
+        private readonly IConnection connection;
+        private readonly IModel channel;
+
+        public RabbitMqService(ILogger<RabbitMqService> logger, MessageSender messageSender)
+        {
+            this.logger = logger;
+
+            this.messageSender = messageSender;
+
+            this.connectionFactory = new ConnectionFactory()
+            {
+                HostName = Environment.GetEnvironmentVariable("RABBITMQ_HOSTNAME") ?? "localhost",
+                UserName = Environment.GetEnvironmentVariable("RABBITMQ_DEFAULT_USER") ?? "localhost",
```

Comment:
Did you mean to put guest/guest for credentials? Looks like localhost snuck in with the refactor 😄

Reply:
I don't actually code. I just copy and paste things 😆.